packages/block-library/src/embed/index.js

```diff
@@ -25,7 +25,7 @@ export const settings = getEmbedBlockSettings( {
 				type: 'raw',
 				isMatch: ( node ) => node.nodeName === 'P' && /^\s*(https?:\/\/\S+)\s*$/i.test( node.textContent ),
 				transform: ( node ) => {
-					return createBlock( 'core/embed', {
+					return createBlock( 'core-embed/paste-handler', {
```

Comment:
Why can't we solve the same issue without using a new temporary paste handler and just consider `core/embed` as the parse handler?

Reply:
Because we can't tell the difference between blocks created by pasting a URL, and blocks loaded from a saved post. The issue is that if we save a post with a working embed, and then we load that post on a day that the embed provider is having problems, we might transform it into a paragraph block when it shouldn't be transformed. Having a dedicated paste handler block makes sure we only ever do that transform on paste.

Reply:
Maybe we should add a comment explaining this (if it's not the case already)

Reply:
I asked the same question here https://github.com/WordPress/gutenberg/pull/11808#discussion_r234580792.

Reply:
> The issue is that if we save a post with a working embed, and then we load that post on a day that the embed provider is having problems, we might transform it into a paragraph block when it shouldn't be transformed. 

What happens if we try to paste at the time when a provider doesn't respond? Does it convert to paragraph, too?